docs/docs/cloud/how-tos/remote_graph.md

```diff
@@ -0,0 +1,150 @@
+# How to interact with the deployment using RemoteGraph
+
+`RemoteGraph` is an interface that allows you to interact with your LangGraph Platform deployment as if it were a regular `CompiledGraph` instance. This guide shows you how you can initialize a `RemoteGraph` and interact with it.
+
+## Setup
+
+When initializing a `RemoteGraph`, you must always specify:
+
+- `name`: the name of the graph you want to interact with. This is the same graph name you use in `langgraph.json` configuration file for your deployment. 
```

Comment:
* Is there a way to list available graphs using the SDK? May be good to specify so (in case people don't know what grahs exist?)

Reply:
no, but maybe we should actually add

Reply:
can do this via `assistants.search()` w/ metadata filter